client/layout/guided-tours/design-showcase-tour.js

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import { translate } from 'i18n-calypso';
+import {
+	overEvery as and,
+	negate as not,
+} from 'lodash';
+import { isDesktop } from 'lib/viewport';
+
+import {
+	makeTour,
+	Tour,
+	Step,
+	ButtonRow,
+	Next,
+	Quit,
+	Continue,
+} from 'layout/guided-tours/config-elements';
+import {
+	inSection,
+	isNewUser,
+	isEnabled,
+	selectedSiteIsCustomizable,
+} from 'state/ui/guided-tours/contexts';
+import Gridicon from 'components/gridicon';
+
+const isAbTestInVariant = function() {};
+
+export const DesignShowcaseTour = makeTour(
+	<Tour
+		name="designShowcase"
+		version="20161123"
+		path="/design"
+		when={ and(
+			isNewUser,
+			isEnabled( 'guided-tours/design-showcase' ),
+			isDesktop,
+			selectedSiteIsCustomizable,
+			not( inSection( 'customize' ) ),
+			isAbTestInVariant( 'designShowcaseTour', 'enabled' ),
+			) }
+		>
+		<Step name="init" placement="right" next="search">
+			<p>
+				{ 'From this page you can change the design of your site. Want to see how to search for your ideal style?' }
+			</p>
+			<ButtonRow>
+				<Next step="search">{ translate( "Let's go!" ) }</Next>
+				<Quit>{ translate( 'No thanks.' ) }</Quit>
```

Comment:
Not sure about that, actually. Since we're both non-native I suggest we'll wait for editorial's opinion here and leave as is for now.